bom/application/pom.xml

```diff
@@ -174,7 +174,7 @@
         <sentry.version>1.7.30</sentry.version>
         <subethasmtp.version>3.1.7</subethasmtp.version>
         <hibernate-quarkus-local-cache.version>0.1.0</hibernate-quarkus-local-cache.version>
-        <kubernetes-client.version>4.13.0</kubernetes-client.version>
+        <kubernetes-client.version>5.0.0-beta-1</kubernetes-client.version>
```

Comment:
```suggestion
        <kubernetes-client.version>5.0.0</kubernetes-client.version>
```

Reply:
Feel free to force push to the PR

Reply:
Force pushed. I think this PR is ready to be merged once CI passes

Reply:
Great, thanks!

I agree, let's merge when CI passes